docs/extendedusage.rst

```diff
@@ -198,6 +200,23 @@ Traits
     'The index of the first track to return'
 
 
+RAML1.0 Types
+~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    >>> api.types
+    {'Person': ObjectType(name='Person', properties={'name': Property(type='string')})}
+    >>> person = api.types['Person']
+    >>> person.type
+    'object'
+    >>> person.description
+    'a Person is a type describing human beings'
+    >>> person.properties
+    {'name': Property(type='string')})
+    >>> person.validate({'foo': 'bar'})
```

Comment:
wouldn't it actually be `validate(raml_file)`? Here this implies that there's a `validate` function attached to the object, which isn't true (and I wouldn't want it).